Formula/tlx.rb

```diff
@@ -0,0 +1,32 @@
+class Tlx < Formula
+  desc "Collection of Sophisticated C++ Data Structures, Algorithms and Helpers"
+  homepage "https://tlx.github.io"
+  url "https://github.com/tlx/tlx/archive/v0.5.20191212.tar.gz"
+  sha256 "5e67d3042a390dbb831b6d46437e3c7fadf738bff362aa7376b210b10ecd532d"
+
+  depends_on "cmake" => :build
+
+  def install
+    args = std_cmake_args + [".."]
+    mkdir "build" do
+      system "cmake", ".", *args
+      system "make", "install"
+    end
```

Comment:
Maybe simple as this?
```suggestion
    system "cmake", "."
    system "make", "install"
```